### PR TITLE
replace all instances of dreditor.org with dreditor.github.io

### PR DESCRIPTION
**Pull request by kevcooper**
dreditor.org is hijacked, this PR replaces all instances of dreditor.org with dreditor.github.io

### CODE REVIEW

**Comment by subhojit777** [comment]
Thanks for the patch. I will be talking to Mark Carver, so that we can replace dreditor.org with dreditor.github.io

**Some background**
dreditor.github.io was made last year when Mark Carver said he can no longer maintain dreditor. Then me and Matthew Grill decided to hold the reigns. I haven't caught up with Mark or Matthew since then. In between I also saw that Mark is rewriting the dreditor. I will talk to him, and finalize on a decision.

**Comment by markhalliwell** [comment]
> dreditor.org is hijacked

Unfortunately, yes. I have known about this for some time, but haven't had the time to do anything about it until today.

The old site is now visible at https://dreditor.unicorn.fail, but key functionality (e.g. "one-click install") isn't likely to be fixed anytime soon given some hard coded aspects of the site.

> Mark Carver said he can no longer maintain dreditor

No. I never said I was no longer maintaining Dreditor. You made that assumption and essentially attempted to hi-jacked this project from me and confuse people.

What I did say was that **browser-based extensions** were no longer maintainable/sustainable and that I would be focusing efforts on integrating non-review specific features directly into d.o and complete rewriting the reviewer as a separate project that could be injected into d.o:

https://www.drupal.org/project/drupalorg/issues/1673278.

This entire org/project needs to be removed.

**Comment by subhojit777** [comment]
> No. I never said I was no longer maintaining Dreditor.

You can check your comments here https://www.drupal.org/project/drupalorg/issues/2779729

image

After reading this, I think anyone would infer that you do not want to maintain Dreditor.

Me and many people have benefitted from this project, and when it came for volunteering to maintain this extension me and Matt Grill stepped up. (although I have done a very bad job as a maintainer). And that is why we created this repository.

--------

> You made that assumption and essentially attempted to hi-jacked this project from me and confuse people.

I never assumed it. It was your comment that made me and many others here https://www.drupal.org/project/drupalorg/issues/2779729 understand that you do not want to maintain this project.

image

This step was never meant as an hijack. I also tried to approach you many times for help via mail and Slack.

image

image

I was asking for help. I don't think a person who is a hijacker would ask for help from an ex-maintainer (now maintainer), if he is really intended to hijack his project.

-----

I was busy with work and another module ([`dc_ajax_add_cart`](https://www.drupal.org/project/dc_ajax_add_cart)), and unfortunately I virtually never got time to put a downloadable link in the project page, nor close the opened issues. I had planned that I will visit this project very soon (in a few months), and fix the dangling ends, like putting a downloadable link of the extension and fix the bugs.

-----

The main project page of dreditor on Drupal https://www.drupal.org/project/dreditor till last year (https://www.drupal.org/node/525726/revisions/10745191/view) it said that the extension has been moved to this repository https://github.com/dreditor/dreditor. Yesterday I found that you have updated link.

Looks like you are going to continue maintain this extension, I will be closing this repository.

**Comment by markhalliwell** [comment]
It's funny how you choose to only focus on the very last little bit of that comment: https://www.drupal.org/project/drupalorg/issues/2779729#comment-11521201

It's also funny how you choose to also _ignore_ my response which was **only 4 days after my previous comment** (mainly due to client work) and just **hours** after you had prematurely forked this project:
https://www.drupal.org/project/drupalorg/issues/2779729#comment-11534501

I asked you _then_ to remove this org/project:

> Re: [#41](https://www.drupal.org/project/drupalorg/issues/2779729#comment-11533953)
> Please take it down. This was premature. I haven't read the comments since only 4 days ago to reconsider the DC Dublin extension.

You also chose to _ignore_ my comment about how you were attempting to hijack the project, that would only result in confusing people:

https://www.drupal.org/project/drupalorg/issues/2779729#comment-11537565

You also chose to _ignore_ my comment explaining what I actually meant by "decommissioning Dreditor".

For some reason, you continue to simply not understand that I was and have **always** referred the EOL of Dreditor within the context of:

**its browser-based extension implementation, NOT THE PROJECT ITSELF**:

https://www.drupal.org/project/drupalorg/issues/2779729#comment-11537943

I even tried talking with you and Matt in IRC back then (sorry, no fancy screenshots to prove it) to discuss why forking the project was premature and how you misunderstood the new direction of the project and to clarify any misunderstandings. Unfortunately, that didn't go over very well from what I remember.

Even after Ryan (mixologic) came up with a brilliant solution, one I have since acknowledged was the correct course of action for everything and everyone involved:
https://www.drupal.org/project/drupalorg/issues/2779729#comment-11538061
https://www.drupal.org/project/drupalorg/issues/2779729#comment-11538445
https://www.drupal.org/project/drupalorg/issues/2779729#comment-11549603
...

You _still continued_ to still keep this fork "alive", ignored everything I had said about this topic and pretend to be the "new maintainer".

> This step was never meant as an hijack. I also tried to approach you many times for help via mail and Slack.

You could have fooled me. As evidence from just the issue that you conveniently and selectively pick apart, you **actively ignored me and everything I had said**.

Why would I waste my time with someone who suddenly, after nearly a year sends me a personal email out of the blue "asking for help" on a project they deemed was "non-mainatained"?



> till last year (https://www.drupal.org/node/525726/revisions/10745191/view) it said that the extension has been moved to this repository https://github.com/dreditor/dreditor. Yesterday I found that you have updated link.

See this is what frustrates me about you. You make claims like this as if you have a leg to stand on and word it in such a way that implies that somehow you have ownership or authority over this project.

The _only_ reason this happened is because _I_ originally had the`dreditor/dreditor` project namespace on GH.

I transferred this project to the new https://github.com/unicorn-fail org when I consolidated all my external projects into a single org (for monitory and permission reasons).

The d.o project page referenced this old namespace and was never updated. Primarily because, before _you_ re-created this org/project, GH automatically redirected it to the new https://github.com/unicorn-fail/dreditor namespace.

Considering that the d.o project was and is an antiquated project, I didn't even realize the d.o project page was now suddenly pointing to the _incorrect_ project until after sun made a change to point to the, again, _incorrect_ domain for this forked project (https://www.drupal.org/node/525726/revisions). This caused the project page to show up in my "Updated" list and I quickly changed to just point to the correct project.



> Looks like you are going to continue maintain this extension

I have been and will continue to be, the primary maintainer the Dreditor project itself, yes.

That has _never_ and _will never_ change.

The **browser-based extensions** are what is no more.

Simply put: the project **evolved** into something different.

Similar to how D8 (with Symfony) is vastly different from D7.

> I will be closing this repository.

Please do. It's confusing people.
